docs/specification/draft/basic/lifecycle.mdx

```diff
@@ -179,6 +180,57 @@ Both parties **SHOULD**:
 - Respect the negotiated protocol version
 - Only use capabilities that were successfully negotiated
 
+#### Resuming Requests
+
+MCP provides a transport-agnostic mechanism for resuming requests across disconnections.
+
+If a client has advertised the [`resumableRequests`][] capability, a server **MAY** send a [`notifications/requests/resumePolicy`][] notification when responding to a request. The notification will specify the resume policy for the request in the event of disconnection, and will include a token that the client can use to resume the request.
+
+After the resume policy is sent, both the client and the server **MAY** disconnect at will. This allows servers to handle long-running requests without maintaining a constant connection.
+
+After a disconnection, clients can resume the request by sending a [`requests/resume`][] request with the **same ID** as the original request, plus the server-issued token as a parameter. If the ID and token are valid per the resume policy, the server **SHOULD** reset policy-related timers, send any pending messages (e.g., progress notifications), and then continue as if it were handling the original request.
```

Comment:
I think the server-issued token should be enough without requiring the request ID. The server can encode the request ID in the token if needed, but I can't think of a case where a client would care about this data. Requiring it seems like a bit of a layer-break between abstractions, previously the RPC calls themselves have never needed to know about the underlying RPC protocol.